frontend/js/global.js

```diff
@@ -0,0 +1,245 @@
+function updateText() {
+	'use strict';
+	var i18n = $.i18n(), language, search, searchFiler, publicDashboard, participantGuidelines, tsdc, thankYouMessage, studyPurpose,
+	dataCollection, dC1, dC2, installNREL, iN1, iN2, iN3, iN4, iN5, joinStudy, jS1, jS2, dataPrivacy, dP1,dP2,dP3,dP4,dP5,dP6,dP7,
+	dP8,dP9,dP10,dP11,dP12,dP13,contact,contactTitle,nrel, about, research, workWithUs, news, careers, contactUs, visit, subscribe, 
+	accessibility, disclaimer, snp, siteFeedback, developers, employees, ft1, usde, oeere, ft2, ase
```

Comment:
I don't doubt that this works, but it seems unnecessarily verbose. It seems like you should be able to use the Data API for all the static text, which will remove the need for this file..
https://github.com/wikimedia/jquery.i18n#data-api

Reply:
Ideally with fallback so that we can more easily see what those texts represent.

Reply:
Yes, I agree that it looks unnecessarily verbose. We where following the example that's in the jquery.i18n GitHub. We will see what we can do to remove this.